src/ovp_pipeline/unified_pipeline_enhanced.py

```
@@ -2565,6 +2574,101 @@ def step_evergreen(self, recent_days: int = 7, dry_run: bool = False, quality_sc
+                result["total_entities"] = before_count
+                return result
+
+            llm_call = None
+            try:
+                from .llm_client import get_litellm_client
+
+                client = get_litellm_client(vault_dir=self.vault_dir)
+                if client:
+                    llm_call = client.call
+            except Exception:
+                pass
```

Ruff (0.15.12)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)